src/components/navigation/navigation-marker/navigation-marker.ts

```diff
@@ -62,7 +63,7 @@ export class SbbNavigationMarkerElement extends SbbNamedSlotListElementMixin<
 
   public select(action: SbbNavigationButtonElement | SbbNavigationLinkElement): void {
     this.reset();
-    action.active = true;
+    toggleDatasetEntry(action, 'actionActive', true);
```

Comment:
This will require a discussion during the refinement, but since the migration to Lit, I have started to just use `this.toggleAttribute('data-action-active', true)` instead of the `toggleDatasetEntry` function.
Thoughts?